src/main/resources/db/migration/V1.0.4__grant_privs_to_reader_user.sql

```diff
@@ -0,0 +1,3 @@
+-- Grant usage and select privs on Reader user
+GRANT USAGE ON SCHEMA wa_workflow_api TO "${dbReaderUsername}";
+GRANT SELECT ON ALL TABLES IN SCHEMA wa_workflow_api TO "${dbReaderUsername}";
```

Comment:
Maybe name this migration so that it specifies it is updating the Single server.
Is the Reader username the same from Single to Flexible server?  Remember AAT is pointing to Flexible server so may need to override the config correctly there.